docs/user/modeling.md

```diff
@@ -0,0 +1,156 @@
+# Modeling
+
+This page describes how to model various kinds of fields on a `nautobot_ssot.contrib.NautobotModel` subclass.
+
+## Quick Reference
+
+The following table describes in brief the different types of model fields and how they are handled.
+
+| Type of field             | Field name                | Notes                                                                                                                                            |
+|---------------------------|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| Normal fields             | Has to match ORM exactly  | Make sure that the name matches the name in the ORM model.                                                                                       |
```

Comment:
Can we break these out by the most specific use case, even if it is repeating itself. I think it should be at least:

- Normal fields on the model
- Custom fields
- django.db.models.OneToOneField
- django.db.models.ForeignKey
- django.db.models.ManyToManyField:
- django.contrib.contenttypes.fields.GenericRelation
- django.contrib.contenttypes.fields.GenericForeignKey

Reply:
didn't follow why this was resolved?

Reply:
I put it into a column in the end, sorry I should have mentioned this.

Reply:
This way I didn't have to duplicate anything, I just mapped it in in the last column

Reply:
The Markdown table looks terrible